docs_manual/source/app_landingzones_create.rst

```diff
@@ -71,3 +71,13 @@ see the zone status and move further with file uploads.
 
 The next sections will provide instructions on browsing your landing zones and
 how to proceed with your file uploads.
+
+
+Landing Zone Update
+^^^^^^^^^^^^^^^^^^^
+
+Landing zones can be updated by clicking the :guilabel:`Update Zone` button on the
```

Comment:
I'm nitpicking now, but the "Update Zone" option within the dropdown menu is not technically a button :) I guess we could call it a "link"?